test/map.js

```diff
@@ -115,7 +170,7 @@ test('first element errors', async (t) => {
     t.fail('must throw')
   } catch (err) {
     // This is 3 in this example
-    t.equal(started > 1, true)
+    t.equal(started === 1, true)
```

Comment:
This should be great then 1. Maybe we should update the test case to add some delay?